src/main/java/com/group7/ecommerce/controller/admin/ManageOrderController.java

```diff
@@ -38,4 +40,14 @@ public String index(
 		model.addAttribute("allStatuses", OrderStatus.values());
 		return "admin/orders/index";
 	}
+
+	@GetMapping("/detail/{id}")
+	public String showOrderDetailPage(@PathVariable("id") Integer orderId, Model model) {
+		OrderDetailResp orderDetail = orderService.getOrderDetailById(orderId);
```

Comment:
```suggestion
		OrderDetailResp order = orderService.getOrderDetailById(orderId);
```

Order là đủ gái ơi :D